source/fem-dem/gls_vans.cc

```diff
@@ -1095,11 +1529,16 @@ GLSVANSSolver<dim>::solve()
     read_dem();
 
   setup_dofs();
+
   this->set_initial_condition(
     this->cfd_dem_simulation_parameters.cfd_parameters.initial_condition->type,
     this->cfd_dem_simulation_parameters.cfd_parameters.restart_parameters
       .restart);
 
+  if (this->cfd_dem_simulation_parameters.void_fraction->mode ==
+      Parameters::VoidFractionMode::qcm)
+    particle_handler.exchange_ghost_particles(true);
```

Comment:
I don't think this needs to be in this if. In all cases, it would be a good idea to always exchange ghost particles when starting the simulation.